tests/integration/icontracts/conftest.py

```
@@ -1,52 +1,14 @@
-
-def mock_llms():
-    env_var = os.getenv("TEST_WITH_MOCK_LLMS", "false")  # default no mocking
-    if env_var == "true":
-        return True
-    return False
-
-
-def pytest_configure(config):
-    load_dotenv(override=True)
+# Import main configuration
+from conftest import *
```

Ruff (0.14.2)

`from conftest import *` used; unable to detect undefined names

(F403)